src/Hwloc.jl

```diff
@@ -207,7 +207,7 @@ end
 type Object
     type_::Symbol
     os_index::Int
-    name::Compat.UTF8String
+    name::Compat.String
```

Comment:
Why is this change necessary?
Will the code still work with Julia 0.5 after this change?
Are you interested in becoming a maintainer of Hwloc.jl?